grouper/models.py

```diff
@@ -791,7 +796,20 @@ def my_log_entries(self):
 
         return AuditLog.get_entries(self.session, on_group_id=self.id, limit=20)
 
+    def my_owners_as_strings(self):
+        """Returns a list of usernames."""
+        return self.my_owners().keys()
+
+    def my_owners(self):
+        """Returns a dictionary from username to records."""
+        od = OrderedDict()
+        for member_type_and_name, member in self.my_members().iteritems():
```

Comment:
nit: you can further unpack this if you'd like. Not a big deal, just FYI:
`for (member_type, name), member in self.my_members().iteritems()`